### PR TITLE
Added support for api-check

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

**Comment by javiercn** [comment]
@pranavkm

File: build/shade/_api-check.shade [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        string[] monikerPrefix,
+        string framework)
+    {
+        var apiListingFile = framework == "net45" ? "baseline.net45.json" : "baseline.netcore.json";
+        var apiListingExceptionsFile = framework == "net45" ? "exceptions.net45.json" : "exceptions.netcore.json";
+
+        var buildFolder = new DirectoryInfo(Path.Combine(directoryInfo.FullName, "bin", configuration));
+
+        FileInfo dll = null;
+        foreach(var prefix in monikerPrefix)
```
Comment (pranavkm) [inline build/shade/_api-check.shade:68]: `foreach (var`

File: build/shade/_api-check.shade [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            return "dotnet";
+        }
+    }
+
+    IList<string> GenerateApiCheckArguments(string framework, params string [] other)
+    {
+        var BASE_DIR = Directory.GetCurrentDirectory();
+        var apiCheckCore = new FileInfo(Path.Combine(BASE_DIR, ".build\\ApiCheck\\tools\\netcoreapp1.0\\ApiCheck.dll"));
+
+        if(framework == "net45")
```
Comment (pranavkm) [inline build/shade/_api-check.shade:45]: `if (`

Comment (pranavkm) [reply build/shade/_api-check.shade:45]: Spacing in a couple of places.

File: build/shade/_k-standard-goals.shade [hunk 3]
```diff
@@ -173,6 +173,9 @@ default SAMPLES_PROJECT_GLOB = "samples/*/project.json"
     }
   }
 
+#run-api-check .build-compile target='compile' description='Checks the compiled assemblies against a baseline for potential breaking changes'
```
Comment (pranavkm) [inline build/shade/_k-standard-goals.shade:176]: 😢 Could we target the `package` target? It would be the same target `NuGetPackageVerifier` runs in.

**Review by pranavkm — APPROVED** [review]

**Pull request merged**